### PR TITLE
Add Public API Baselines for JSInterop, Localization, Logging.AzureAppServices

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

**Comment by dougbu** [comment]
Need to run a quick errand but I'll approve this when I get back&hellip;

File: AspNetCore.sln [hunk 1]
```diff
@@ -1506,6 +1507,22 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FilePr
...
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Localization.Tests", "src\Localization\Localization\test\Microsoft.Extensions.Localization.Tests\Microsoft.Extensions.Localization.Tests.csproj", "{16B899B4-A4F4-4EF7-93BB-355861977A12}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Logging.AzureAppServices", "Logging.AzureAppServices", "{3EAB9890-2C01-444C-ACA0-D77B29CDE08B}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.AzureAppServices", "src\Logging.AzureAppServices\src\Microsoft.Extensions.Logging.AzureAppServices.csproj", "{3E29454A-C4DC-44B7-AF0A-A782AD2E73BC}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.AzureAppServices.Tests", "src\Logging.AzureAppServices\test\Microsoft.Extensions.Logging.AzureAppServices.Tests.csproj", "{43E3B132-2486-44A3-92C6-39E39724FAFD}"
+EndProject
```
Comment (dougbu) [inline AspNetCore.sln:1525]: Should we worry that this solution is missing some implementation projects❔ I lean toward ignoring it and continuing to add projects as people need to make changes in the less-touched areas.

File: src/Logging.AzureAppServices/src/BlobLoggerProvider.cs [hunk 2]
```diff
@@ -29,6 +30,7 @@ public class BlobLoggerProvider : BatchingLoggerProvider
         /// Creates a new instance of <see cref="BlobLoggerProvider"/>
         /// </summary>
         /// <param name="options">The options to use when creating a provider.</param>
+        [SuppressMessage("ApiDesign", "RS0022:Constructor make noninheritable base class inheritable", Justification = "Required for backwards compatibility")]
```
Comment (dougbu) [inline src/Logging.AzureAppServices/src/BlobLoggerProvider.cs:33]: Kind-of like this check in the analyzer 🥇

**Review by dougbu — APPROVED** [review]
Side question: Have you hit any projects needing `$(MSBuildProjectDirectory)\$(_TFMDirectory)\PublicAPI.Shipped.txt` files❔ I know we have a few `#if` blocks that change the public API but haven't hit that case yet. Then again, I haven't built my lower-level baseline addition branch yet&hellip;

File: AspNetCore.sln (hunk 1 above)
Comment (Pilchie) [reply AspNetCore.sln:1525]: I'm not that worried about it, until we start using it to actually build the product ;)

Comment (Pilchie) [reply AspNetCore.sln:1525]: I suspect that none of the projects that came over from extensions got added.

**Comment by Pilchie** [comment]
Personally I haven't needed TFM specific public API yet.

**Pull request merged**